Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Default ignored files\r\n/shelf/\r\n/.idea/workspace.xml\r\n.idea/
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	
+++ b/.gitignore	
@@ -1,4 +1,5 @@
 # Default ignored files
 /shelf/
 /.idea/workspace.xml
-.idea/
\ No newline at end of file
+.idea/
+out/
\ No newline at end of file
